File: LV1/콜라츠 추측.java
class Solution {

  public int solution(int num) {
    for (int count = 0; count < 500; count++) {
      if (num == 1) {
        return count;
      }
      num = num % 2 == 0 ? num / 2 : num * 3 + 1;
    }
    return -1;
  }
}

File: LV1/콜라츠 추측.js
function solution(num) {
  for (let count = 0; count < 500; count++) {
    if (num === 1) {
      return count;
    }
    num = num % 2 === 0 ? num / 2 : num * 3 + 1;
  }
  return -1;
}
